website/content/en/guides/developer/debugging/debugging.md

```diff
@@ -0,0 +1,760 @@
+---
```

Comment:
This seems like the type of doc that the Documentation team could really help refine 🙏